tools/mo/openvino/tools/mo/main.py

```diff
@@ -120,7 +120,7 @@ def get_moc_frontends(argv: argparse.Namespace):
     if not fem or use_legacy_frontend:
         return None, []
 
-    available_moc_front_ends = fem.get_available_front_ends()
+    available_moc_front_ends = get_available_front_ends()
```

Comment:
Shouldn't you pass `fem` to this function?

Reply:
Great catch. Done!